pages/howto/easybuild.md

```diff
@@ -102,9 +102,9 @@ These are automatically loaded for you when you load the `foss/2021b` module.  W
 **When using Easybuild, do NOT use the CCR login nodes.**  Always use a [compile node](../software/building.md) or do this from a compute node in an [OnDemand desktop](../portals/ood.md#desktop-interactive-apps) session or [interactive job](../hpc/jobs.md#interactive-job-submission).  These installations can use a decent amount of disk space so we recommend you use your project directory for all software installations.  We do not recommend using your home directory as these have small quotas.  If you haven't already done so, create an `easybuild` directory in your group's project directory and set the [Easybuild build prefix](../software/building.md#building-modules-for-your-group) to that directory.  Reminder: this is just an example; you'll need to set the path name to your group's shared project directory, which may not be in `/projects/academic`      
 
 ```
-$ ssh compile  
-$ mkdir /projects/academic/groupname/easybuild   
-$ export CCR_BUILD_PREFIX=/projects/academic/groupname/easybuild  
+CCRusername@login ~:$ ssh compile  
```

Comment:
`CCRusername@login:~$`

Change ` ~:$` to `:~$` for lines 105-107.

Reply:
Fixed this and also changed the look of the login for the compile node throughout this page as it's slightly different than the login nodes.